flux/values.yaml

```diff
@@ -0,0 +1,63 @@
+flux:
+  rbac:
+    create: false
+    pspEnabled: false
+  serviceAccount:
+    create: false
+  clusterRole:
+    create: false
+  registry:
+    dockercfg:
+      enabled: true
+      secretName: "okteto-regcred"
+      configFileName: /dockercfg/config.json
+  extraContainers:
+    - name: recv
+      image: fluxcd/flux-recv:0.4.0
+      args:
+      - --config=/etc/fluxrecv/fluxrecv.yaml
+      ports:
+      - name: recv
+        containerPort: 8080
+        protocol: TCP
+      resources:
+        limits:
+          cpu: 100m
+          memory: 200Mi
+      readinessProbe:
+        httpGet:
+          path: /health
+          port: 8080
+      volumeMounts:
+      - name: fluxrecv-config
+        mountPath: /etc/fluxrecv
+  extraVolumes:
+    - name: fluxrecv-config
+      secret:
+        secretName: fluxrecv-config
+        defaultMode: 0400
+  resources:  
+    limits:
+      cpu: 100m
+      memory: 500Mi
+  memcached:
+    resources:
+      limits:
+        cpu: 100m
+        memory: 628Mi
```

Comment:
does it need that much memory? maybe put a lot less, so it leaves a lot of memory for the actual appllications? (or is there a way to run it without it?